include/sys/zio_impl.h

```diff
@@ -24,7 +24,7 @@
  */
 
 /*
- * Copyright (c) 2013 by Delphix. All rights reserved.
+ * Copyright (c) 2012, 2015 by Delphix. All rights reserved.
```

Comment:
Thanks. I see no reason to object to a copyright holder changing their own copyright line.